packages/datadog-plugin-cucumber/test/index.spec.js

```diff
@@ -125,13 +146,16 @@ describe('Plugin', () => {
           errors
         } = test
 
-        describe(`for ${featureName}${featureSuffix}`, () => {
-          it('should create a test span', async () => {
+        describe(`for ${featureName}${featureLineNumber}`, () => {
+          it('should create a test span and spans for integrations', async function () {
+            this.timeout(200000)
```

Comment:
Why such a high timeout?

Reply:
this is a left over from when I was testing. It shouldn't be needed. I'll remove it

Reply:
removed in https://github.com/DataDog/dd-trace-js/pull/1406/commits/e1a029248ba60bcfa057a7ca7cd20c8f870decca